EDEngineer/Resources/Data/localization.json

```diff
@@ -1718,12 +1718,6 @@
       "de": null,
       "fr": null,
       "ru": null
-    },

```

Comment:
yes you just need to add back these lines you delete (the block "Select Language")